docs/spec/src/integration/spec/6-secure-integration.md

```diff
@@ -0,0 +1,65 @@
+# Secure Integration
+
+This page is meant to be read by eigenda and rollup developers who are writing a secure integration. To understand what a secure integration is at a high level, visit our [secure integration overview](https://docs.eigenda.xyz/integrations-guides/rollup-guides/integrations-overview) docs instead.
+
+## Cert+Blob+Timing Validation
+
+Blob and Cert verification is done for both the write (sequencer) and read (validator) paths. Given this duplication in the lifecycle, and given its importance, it deserves its own section.
+
+The validation process is:
+
+1. Validate the Cert (against state on ethereum)
+2. Validate the Blob (against the Cert)
+
+### Cert Validation
+
+Cert validation is done inside the EigenDACertVerifier contract, which EigenDA deploys as-is, but is also available for rollups to modify and deploy on their own. Specifically, the [verifyDACertV2](https://github.com/Layr-Labs/eigenda/blob/98e21397e3471d170f3131549cdbc7113c0cdfaf/contracts/src/core/EigenDACertVerifier.sol#L86) is the entry point for validation. This could either be called during a normal eth transaction (either for pessimistic “bridging” like EigenDA V1 used to do, or when uploading a Blob Field Element to a one-step-proof’s [preimage contract](https://specs.optimism.io/fault-proof/index.html#pre-image-oracle)), or be zk proven using a library like [Steel](https://github.com/risc0/risc0-ethereum/blob/main/crates/steel/docs/what-is-steel.md).
```

Comment:
> Specifically, the verifyDACertV2 is the entry point for validation.

Should either be "Specifically, ~~the~~ verifyDACertV2 is the entry point for validation.", or "Specifically, the verifyDACertV2 *function* is the entry point for validation."

Reply:
[176dd65](https://github.com/Layr-Labs/eigenda/pull/1439/commits/176dd652e57e14075b2e32d93af34488a4fff2d9)